tests/test_api.py

```diff
@@ -446,12 +446,13 @@ def test_preprocess_error(self):
         processor = api.GradeCSVProcessor(course_id=self.course_id)
         row = {
             'block_id': self.usage_key,
+            'new_override-123402db': '1',
```

Comment:
Glad that current tests are updated but would also argue for a test that calls out fixed behavior, e.g. do a bunch of overrides on just one user, make sure the update grade API is getting called with the right grades/subsections.